docs/zephyr-integration.md

```diff
@@ -0,0 +1,493 @@
+  "projectKey": "PROJ"
+}
+```
+
+**Use Cases:**
+- Check which test cases cover a specific bug or feature
+- Identify gaps in test coverage for requirements
+- Generate coverage reports for stakeholders
+
+### Test Case Management
+
+Complete CRUD operations for test case lifecycle management.
+
+#### `zephyr_list_test_cases`
+
+List test cases for a project.
+
+**Parameters:**
+- `projectKey` (required): JIRA project key
+
+#### `zephyr_list_test_cases__nextgen_`
+
+List test cases using NextGen API with enhanced filtering.
+
+**Parameters:**
+- `projectKey` (required): JIRA project key
+- `folderId` (optional): Folder ID to filter by
+- `maxResults` (optional): Maximum number of results (default: 50, max: 100)
+
+#### `zephyr_get_test_case`
+
+Get details of a specific test case.
+
+**Parameters:**
+- `testCaseKey` (required): Test case key in Zephyr format
+
+#### `zephyr_create_test_case`
+
+Create a new test case with comprehensive metadata.
+
+**Parameters:**
+- `testCaseData` (required): Object containing test case information
+  - `name` (required): Test case name
+  - `projectKey` (required): JIRA project key
+  - `objective` (optional): Test case objective
+  - `precondition` (optional): Test precondition
+  - `estimatedTime` (optional): Estimated time in minutes
+  - `componentId` (optional): Component ID
+  - `priorityName` (optional): Priority name
+  - `statusName` (optional): Status name
+  - `folderId` (optional): Folder ID
+  - `ownerId` (optional): Owner ID
+  - `labels` (optional): Array of test case labels
+  - `customFields` (optional): Custom fields object
+
+**Example:**
+```json
+{
+  "testCaseData": {
+    "name": "User Login Validation",
+    "projectKey": "PROJ",
+    "objective": "Verify user can login with valid credentials",
+    "folderId": 123,
+    "priorityName": "High"
+  }
+}
+```
+
+#### `zephyr_update_test_case`
+
+Update existing test case properties.
+
+**Parameters:**
+- `testCaseKey` (required): Test case key in Zephyr format (e.g., PROJECT-T123)
+- `updateData` (required): Object containing fields to update
+  - `name` (optional): Updated test case name
+  - `objective` (optional): Updated test case objective
+  - `precondition` (optional): Updated test precondition
+  - `estimatedTime` (optional): Updated estimated time in minutes
+  - `componentId` (optional): Updated component ID
+  - `priorityName` (optional): Updated priority name
+  - `statusName` (optional): Updated status name
+  - `folderId` (optional): Updated folder ID
+  - `ownerId` (optional): Updated owner ID
+  - `labels` (optional): Updated test case labels
+  - `customFields` (optional): Updated custom fields
+
+#### `zephyr_add_test_script`
+
+Add executable script or detailed instructions to test case.
+
+**Parameters:**
+- `testCaseKey` (required): Test case key in Zephyr format (e.g., PROJECT-T123)
+- `text` (required): Test script content
+- `type` (optional): Test script type - must be 'plain' or 'bdd'. Defaults to 'plain'.
```

Comment:
```suggestion
- `type` (optional): Test script type - must be 'plain' or 'bdd'. Defaults to 'plain'. To create a step-by-step test script, you should first create a plain text and then add the steps
```

Reply:
Remember that these comments aren't to be read by humans, the MCP will use AI to interpret this. With that in mind, is there more context you would like to provide here?

Reply:
It does not look like that... 
```
├── src/<product-name>/      # Specific SmartBear product integration
├── src/common/              # Shared utilities and types
├── src/tests/               # Test files
├── docs/                    # Documentation for developer.smartbear.com site
└── scripts/                 # Build and utility scripts
```
this will be part of a public documentation page under https://developer.smartbear.com/smartbear-mcp/docs/mcp-server.
This zephyr-integration should be linked on the manifest.json please add it there, so that it will be shown under `MCP SERVER Capabilities`